### PR TITLE
spring boot controler+service+dao

### PR DESCRIPTION
**Pull request by un-install**

### CODE REVIEW

**Review by Daniluko — APPROVED** [review]
where your entity?

**Comment by un-install** [comment]
they are in the master

File: src/main/java/boot/config/SwggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwggerConfig implements WebMvcConfigurer {
+
```
Comment (SerhiiPasko) [inline src/main/java/boot/config/SwggerConfig.java:13]: What about @Bean Docket? Your swagger work?

**Review by SerhiiPasko — APPROVED** [review]
Read comment. I think your swagger don't working

File: src/main/java/boot/config/SwggerConfig.java (hunk 1 above)
Comment (un-install) [reply src/main/java/boot/config/SwggerConfig.java:13]: Yes it is working

**Pull request merged**